install/Install_Controller.php

```diff
@@ -783,10 +783,6 @@ private function action_databasePopulation()
 		}
 		$replaces['{$default_reserved_names}'] = strtr($replaces['{$default_reserved_names}'], array('\\\\n' => '\\n'));
 
-		// If the UTF-8 setting was enabled, add it to the table definitions.
-		if (!empty($databases[$db_type]['utf8_support']))
-			$replaces[') ENGINE=MyISAM;'] = ') ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci;';
```

Comment:
True, but the same answer I gave at https://github.com/emanuele45/Dialogo/commit/bb16cdc4e74435a5f44aced379d0e169a568edfc#commitcomment-19058260 applies.